index.html

```diff
@@ -0,0 +1,42 @@
+            alt="beauty hotels" id="header-img">
+        <nav id="nav-bar">
+            <ul class="nav-link">
+                <li>
+                    <a class="nav-link" href="#Home"></a> Home</li>
+                <li>
+                    <a class="nav-link" href="#About us"></a> About Us</li>
+                <li>
+                    <a class="nav-link" href="#contact us"></a> Contact Us</li>
+                <!-- <li>
+                    <a class="nav-link" href="#contact us"></a> Contact Us</li> -->
+            </ul>
+
+        </nav>
+        <embed src="" type=" " id="video ">
+
+    </header>
+    <form action="https://www.freecodecamp.com/email-submit)" id="form">
+        <label for="email ">Enter Email:</label>
+        <input type="email" id="email" placeholder="#email" required>
+        <input type="submit" id="submit">
```

Comment:
Kindly use the correct tag which is a button. Using input as a tag with the type Submit, doesn't make sens.

You can simply use:
 <button></button>

Reply:
- [ ] I'm seeing some really professional comments here well done 👏. But please make sure you're comments are based on facts or personal experience as the comment you made above is a bit ambiguous. You can definitely use the `input` element with the `type` set to `submit`. If you have further evidence that doesn't support this I advise putting the link so the student can refer to it. ✔
    - So please consider removing the above comment if you can't validate it.
- [ ] It would also be better if you could use emojis in your comments as they'll make it more human-friendly ☺

Reply:
- [ ] I'm seeing some really professional comments here well done 👏. But please make sure you're comments are based on facts or personal experience as the comment you made above is a bit ambiguous. You can definitely use the `input` element with the `type` set to `submit`. If you have further evidence that doesn't support this I advise putting the link so the student can refer to it. ✔
    - So please consider removing the above comment if you can't validate it.
- [ ] It would also be better if you could use emojis in your comments as they'll make it more human-friendly ☺